dev/src/generate_sql_schema.py

```diff
@@ -883,6 +915,34 @@ def get_on_action_mode(action: str, delete: bool) -> str:
                 raise Exception(f"{action} is not a valid action mode")
         return ""
 
+    @staticmethod
+    def get_foreign_key_notify_trigger(
+        table_name: str,
+        foreign_table: str,
+        ref_column: str,
+        fk_columns: list[str] | str,
+        initially_deferred: bool = False,
+        delete_action: str = "",
+        update_action: str = "",
+    ) -> str:
+        FOREIGN_KEY_NOTIFY_TRIGGER_TEMPLATE = string.Template(
+            "CREATE TRIGGER ${trigger_name} AFTER INSERT OR UPDATE OF ${ref_column} OR DELETE ON ${own_table}\n"
+            + "FOR EACH ROW EXECUTE FUNCTION log_modified_related_models('${foreign_table}', '${ref_column}');\n"
+        )
+
+        trigger_name = HelperGetNames.get_notify_related_trigger_name(
+            table_name, ref_column
+        )
+        own_table = HelperGetNames.get_table_name(table_name)
+        return FOREIGN_KEY_NOTIFY_TRIGGER_TEMPLATE.substitute(
+            {
+                "trigger_name": trigger_name,
+                "own_table": own_table,
+                "ref_column": ref_column,
+                "foreign_table": foreign_table,
+            }
+        )
```

Comment:
I think this is more readable and shorter since you can delete lines 928 through 932 also and the concatenation will happen in one place.
```suggestion
        return f"CREATE TRIGGER {trigger_name} AFTER INSERT OR UPDATE OF {ref_column} OR DELETE ON {own_table}\n FOR EACH ROW EXECUTE FUNCTION log_modified_related_models('{foreign_table}', '{ref_column}');\n"
```